### ISSUE TITLE
Eagles files for the mainboard wrong

### ISSUE THREAD

**Issue by NoSkillsToPayTheBillz** [x0xb0x/x0xb0x.github.io#14]
![sloppy outline](https://user-images.githubusercontent.com/51103107/68996837-81c74880-086d-11ea-82a8-7f5d9b6291a4.jpg)

The .Brd files for the main board say they were made in eagle 6.4 the originals are from eagle 4.13 or so it says on the original download page. Either way they are edited and it doesn't have a outline and looks like some one tried to added on very sloppy on layer 20 and there is a few air-wire errors and alot of unconnected wires. I don't know if they are supposed to be like that. But the outline makes it hard to process the files. I didn't no if this goes here or not but since there was a upload not to long ago of the files i figured some one might be able to fix it.

**Comment by ghost** [comment]
The files I was able to locate online are corrupt and don't load into Eagle?
Are you able to share the files you found?

**Comment by NoSkillsToPayTheBillz** [comment]
i used these files someone posted on a forum 
[mainboard.zip](https://github.com/x0xb0x/x0xb0x.github.io/files/4394515/mainboard.zip)
[io-board.zip](https://github.com/x0xb0x/x0xb0x.github.io/files/4394517/io-board.zip)

**Comment by ghost** [comment]
Thanks so much... I see the artifacts, but no BRD file? 
Eagle can open the GBL, but only as a text file?
I also have the SCH file file, but this only shows schematic, not the v1 board layout.

**Comment by ghost** [comment]
KiCAD is able to open the Gerbers, so I can confirm that the board loads and 'looks complete' to my untrained eye. Has anyone actually has the PCBs fabricated from these files and built a working x0xb0x?

**Comment by reinvantveer** [comment]
No, they come unaltered from the original site at http://www.ladyada.net/make/x0xb0x/

**Comment by reinvantveer** [comment]
But you could ask a manufacturer, I ordered a kit  from a company in Germany long time ago. Adafruit sells a board as well, on https://www.adafruit.com/product/3603 so you could ask around whether they did any updates on the board files

**Comment by ghost** [comment]
@reinvantveer thanks for confirming...
